modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -174,6 +175,42 @@ for login credentials:
 * Log-in credentials, save in database
 * Global credentials
 
+==== Global Credentials
+
+To authenticate with global credentials:
+
+* *The username is the same as the user id of the ownCloud session.* 
+  For example, the ownCloud user "Alice" with the password `mysecret` (or even without a password) uses `Alice` as their username to obtain access to the WND storage. 
++
+NOTE: For LDAP, the user id is usually `1223-cbdf-cacc-1234...`, unless the configuration in {ldap-app-url}[the user_ldap app] is changed.
```

Comment:
is it worthy to point what they need to change here? I mean, I guess this point to the user_ldap configuration, but the admin can start touching things randomly.

Reply:
Well said. I removed that line.